version/version.go

```diff
@@ -19,9 +19,14 @@ import (
 	"runtime"
 )
 
+//var needs to be used instead of const for ldflags
 var (
-	Version    = "v0.14.0+git"
-	GitVersion = "unknown"
-	GitCommit  = "unknown"
-	GoVersion  = fmt.Sprintf("%s %s/%s", runtime.Version(), runtime.GOOS, runtime.GOARCH)
+	Version                 = "v0.14.0+git"
+	GitVersion              = "unknown"
+	GitCommit               = "unknown"
+	KubernetesVendorVersion = "unknown"
```

Comment:
Since we technically support a range of k8s versions, it would be informative to print min and max k8s version variables:
```console
$ operator-sdk version
operator-sdk version: "v0.14.0", commit: ..., minKubernetesVersion: "v1.14.0", maxKubernetesVersion: "v1.16.3"
```
WDYT?

Reply:
@hasbro17 @joelanford ^

Reply:
Yes that was the intent. The version from the client-go `kubernetes-x.y.z` tag plus a link is a better approach to capture this information.

Reply:
We technically do support a range of k8s versions (i.e client-go matrix) but we don't really test against supposedly compatible k8s versions.

So I agree that linking to the client-go matrix as an indirect indication of compatibility is okay but probably not listing out the exact versions here since we haven't tested and guaranteed that the SDK works against all of them.

Reply:
So right now this would just be the k8s version in our CI [`K8S_VERSION`](https://github.com/operator-framework/operator-sdk/blob/master/hack/ci/setup-k8s.sh#L8).

Reply:
Yep, that sounds good to me.

Ideally (possibly for a follow-on) we would enforce in CI that K8S_VERSION aligns with our client-go dependency. How doable would that be?